harbor/datadog_checks/harbor/data/conf.yaml.example

```diff
@@ -1,99 +1,348 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
 
     ## @param url - string - required
     ## The Harbor HTTP url.
     #
-  - url: http://<HARBOR_URL>
+  - url: <URL>
```

Comment:
I don't know if having `http://<URL>` as an example is helpful

Reply:
doesn't seem like we have a default https://github.com/DataDog/integrations-core/blob/master/harbor/datadog_checks/harbor/harbor.py#L106